tests-legacy/Unit/ContextMocker.php

```diff
@@ -111,10 +112,9 @@ public function mockContext()
         Module::setContextInstanceForTesting($context);
         $context->shop = new Shop((int) Configuration::get('PS_SHOP_DEFAULT'));
         Shop::setContext(Shop::CONTEXT_SHOP, (int) Context::getContext()->shop->id);
-        $context->customer = Phake::mock('Customer');
-        Phake::when($context->customer)->getGroups()->thenReturn(array());
-        $context->cookie   = Phake::mock('Cookie');
-        $context->country  = Phake::mock('Country');
+        $context->customer = new Customer();
```

Comment:
Hum, it's not a mock anymore :thinking: 
It now relies on the database if something is trigger